tools/on_device_tests_gateway.proto

```diff
@@ -52,10 +49,14 @@ message OnDeviceTestsCommand {
   string unittest_shard_index = 17;
   string test_attempts = 18;
   string retry_level = 19;
+  string start_timeout = 20;
+  string test_timeout = 21;
+  string builder_url = 22;
```

Comment:
Don't merge until corresponding change is deployed on the server side

Reply:
Has it been deployeth ?